src/OLEDDisplay.h

```diff
@@ -139,7 +139,8 @@ enum OLEDDISPLAY_GEOMETRY {
   GEOMETRY_128_64   = 0,
   GEOMETRY_128_32   = 1,
   GEOMETRY_64_48    = 2,
-  GEOMETRY_RAWMODE  = 4
+  GEOMETRY_RAWMODE  = 4,
+  GEOMETRY_64_32    = 5
```

Comment:
Hmm, shouldn't this be placed _before_ the raw mode and id 3 instead of 5?

Reply:
ok, I changed this, please check.